src/docfx/Models/BuildJsonConfig.cs

```diff
@@ -96,5 +58,9 @@ public class BuildJsonConfig
         /// </summary>
         [JsonProperty("noLangKeyword")]
         public bool NoLangKeyword { get; set; }
+
+        [JsonProperty("docsets")]
+        public List<BuildJsonConfigCommon> Docsets { get; set; }
```

Comment:
As I only add a property "docset" property, it's compatible with the old docfx.json.